db/migrate/20190726204302_remove_local_default_embedded_ansible_repos.rb

```diff
@@ -0,0 +1,20 @@
+class RemoveLocalDefaultEmbeddedAnsibleRepos < ActiveRecord::Migration[5.1]
+  class ConfigurationScriptSource < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  class GitRepository < ActiveRecord::Base; end
+
+  CONFIGURATION_SCRIPT_TYPE = "ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScriptSource".freeze
+
+  def up
+    css = ConfigurationScriptSource.where(:type => CONFIGURATION_SCRIPT_TYPE).where("scm_url LIKE ? OR scm_url = ?", "%/content/ansible_consolidated", "file:///var/lib/awx_consolidated_source")
+
+    css.each do |configuration_script|
+      git_repo = GitRepository.find_by(:id => configuration_script.git_repository_id)
```

Comment:
Should we do this in bulk instead of instantiating every `css` object?

`GitRepository.where(:id => css.pluck(:id)).delete_all`